client/Components/BookATourForm.js

```diff
@@ -1,8 +1,139 @@
+    if (currentNumber === 1 && direction === 'down') return;
+    direction === 'up' ? currentNumber += 1 : currentNumber -=1;
+    input.value = currentNumber;
+    changeNumberOfPeople(currentNumber);
+  }
 
-const BookATourForm = () => {
   return (
-    <h1>Book Tour form here</h1>
+    <div className="get-in-touch-form-wrapper">
+    <button className="close-form" onClick={() => { handleFormChange('self-serve')}}>X</button>
+    <h3>Book a tour</h3>
+
+    <form>
+      <p>Your health and safety is important to us. That’s why we introduced <a href="#" style={{color: 'blue'}}> touring guidelines</a> to ensure you feel comfortable and confident visiting our spaces.</p>
+      <div className="form-item">
+        <div className="field" >
+          <label htmlFor="fullName">Full name</label>
+          <input
+            id="fullName"
+            name="fullName"
+            type="text"
+            required
+            onFocus={handleFormFieldFocus}
+            onBlur={(e) => {handleFieldBlur(e, e.target.name)}}/>
+            <div className="error-message" hidden={isHidden.fullName}>First and last, please.</div>
+        </div>
+      </div>
+      <div className="form-item">
+        <div className="field">
+          <label htmlFor="email">Work Email</label>
+          <input
+            id="email"
+            name="email"
+            type="email"
+            required
+            onClick={handleFormFieldFocus}
+            onBlur={(e) => {handleFieldBlur(e, e.target.name)}}/>
+            <div className="error-message" hidden={isHidden.email}>We need this for confirmations.</div>
+        </div>
+      </div>
+      <div className="form-item">
+        <div className="field">
+          <label htmlFor="phone">Phone Number</label>
+          <input
+            id="phone"
+            name="phone"
+            type="phone"
+            required
+            onClick={handleFormFieldFocus}
+            onBlur={(e) => {handleFieldBlur(e, e.target.name)}}/>
+            <div className="error-message" hidden={isHidden.phone}>Please give us a way to get in contact.</div>
+        </div>
+      </div>
+      <div className="cell-container">
+        <div className="form-item cell">
+          <div className="cell-wrap">
+            <label htmlFor="moveInDateBtn">Move-in date</label>
+            <button id="moveInDataBtn" type="button" className="move-in" onClick={handleMoveInClick}>Jan 21</button>
+          </div>
+          <input id="calendar" type="date" hidden/>
+        </div>
+        <div className="form-item cell">
+          <div className="cell-wrap">
+            <label htmlFor="desiredCapacity">Number of People</label>
+            <input
+              id="desiredCapacity"
+              type="number"
+              defaultValue="1"
+              required></input>
+            <button className="down" onClick={handlePeopleChange}>-</button>
+            <button className="up" onClick={handlePeopleChange}>+</button>
```

Comment:
an alternative to using e.preventDefault is adding type=button attr to buttons inside a form to prevent the default submit behavor

Reply:
I didn't know this! also didn't realize a button without a type is assumed as a 'submit'